docs/src/main/asciidoc/cyclonedx.adoc

```diff
@@ -0,0 +1,183 @@
+
+* `<executable-name>` is the base file name (without the extension) of the executable that launches an application;
+* `<format>` is either `json` (the default) or `xml`, which can be configured using `quarkus.cyclonedx.format` property. If both formats are desired `quarkus.cyclonedx.format` can be set to `all`.
+
+=== Uber JAR
+
+SBOMs for Uber JARs will use the Uber JAR Maven artifact as their main component.
+
+Since an Uber JAR is a Maven artifact itself, SBOMs generated for Uber JARs will also be automatically attached as project artifacts by the Quarkus Maven plugin. This, however, can be disabled by setting `attachSboms` parameter of the `quarkus:build` goal to `false`.
+
+Gradle users will have to explicitly configure a publishing plugin to deploy SBOMs as Maven artifacts.
+
+=== Native image
+
+SBOMs for native images will use the native executable file as their main component.
+
+=== Fast JAR
+
+Fast JAR packaging uses a Quarkus-specific filesystem directory layout that contains files generated by Quarkus and Maven artifacts that are runtime dependencies of an application.
+
+SBOMs for Fast JAR packaging type will use the executable JAR file as their main component and record both runtime and build time Quarkus application dependencies.
+
+==== Runtime Components
+
+Every file in the resulting Fast JAR distribution will appear in the SBOM with `quarkus:component:scope` property set to `runtime` and `evidence.occurences.location` field pointing to the location of the component in the application distribution directory, for example
+
+[source,json]
+----
+      "purl" : "pkg:maven/org.jboss.slf4j/slf4j-jboss-logmanager@2.0.0.Final?type=jar",
+      "properties" : [
+        {
+          "name" : "quarkus:component:scope",
+          "value" : "runtime"
+        }
+      ],
+      "evidence" : {
+        "occurrences" : [
+          {
+            "location" : "lib/main/org.jboss.slf4j.slf4j-jboss-logmanager-2.0.0.Final.jar"
+          }
+        ]
+      }
+----
+
+NOTE: `evidence.occurrences.location` was introduced in CycloneDX schema version 1.5, for older versions the location will be indicated using `quarkus:component:location` property.
+
+==== Pedigree
+
+Pedigree is a way to provide information that certain patches, or changes in general, have been applied to a certain component.
+
+In certain cases Quarkus may copy modified versions of dependency artifacts to an application distribution. Manipulating original content of a component will change its hash sums which may get highlighted as suspicious by the tools comparing original component hash sums to those found in the distribution.
+
+When Quarkus applies modifications to artifacts resolved from Maven repositories it can manifest these changes as pedigree notes in the generated SBOM.
```

Comment:
```suggestion
When Quarkus applies modifications to artifacts resolved from Maven repositories, it can manifest these changes as pedigree notes in the generated SBOM.
```